### ISSUE TITLE
Missing name/email address from SSO

### ISSUE THREAD

**Issue by bartaz** [canonical-ols/javan-rhino#116]
If user doesn't share his name and/or email address from SSO UI has missing info:

<img width="948" alt="screen shot 2016-09-30 at 13 45 43" src="https://cloud.githubusercontent.com/assets/83575/18990749/4e74d2f0-8714-11e6-8c63-09cb2d705e6c.png">

Not sure if it's just staging SSO issue, or can we make name (and email?) required?
If we don't want to require those, UI should not expect them and adapt text.

**Comment by earnubs** [comment]
In `server/openid/relyingparty.js`: 

```
new openid.SimpleRegistration({
  'email' : 'required',
  'fullname' : 'required'
}),
```

**Comment by earnubs** [comment]
MU needs to become a trusted RP in SSO, in order to properly support `required` sreg attrs, which I'll talk to @canonical-ols/ols about today

**Comment by bartaz** [comment]
Fixed by #125 #130

**Comment by bartaz** [comment]
Also thanks to trusted RP email and name are properly required now.
